README.md

```diff
@@ -85,13 +82,21 @@ def getGithubSecret():
 
 def getSecretKey():
     return 'DJANGOSECRETXXX'
+
+def getOauthToolKitAppID():
+    return 'OauthAppIDXXX'
+
+def getOauthToolKitAppSecret():
+    return 'OauthAppSecretXXX'
 ```
 
 where:
 
 * GHKEYXXX is the Client ID for the Github Oauth Apps
-* GHSECRETXXX is the Client ID for the Github Oauth Apps
+* GHSECRETXXX is the Client secret for the Github Oauth Apps
 * DJANGOSECRETXXX is the Django secret
+* OauthAppIDXXX is the client ID of the django oauth toolkit app
```

Comment:
We can add a note here or add a link to the below section where it is described how to setup the django oauth app, otherwise users may get confused.